wolox-books/src/app/app.state.ts

```diff
@@ -1,6 +1,6 @@
 import { Book } from './screens/book/book';
 
 export interface AppState {
-  readonly book: Book[];
+  readonly book: Object;
```

Comment:
because when I show the book, this saves the state like `{ books: [books listed], book: Object{book to show}` and that converts the book in an Object type.

Reply:
It'd be nice to define that type in that case